caffe2/python/scope.py

```diff
@@ -7,7 +7,6 @@
 
 import contextlib
 import threading
-from past.builtins import basestring
```

Comment:
there is usage of `basestring` in this same file
https://github.com/pytorch/pytorch/blob/226ff68475b2a7b85e2277e69c3a68430c41aca8/caffe2/python/scope.py#L37